l3backend/l3backend-graphics.dtx

```diff
@@ -61,20 +61,28 @@
 %    \end{macrocode}
 %
 % \begin{macro}{\@@_backend_loaded:n}
+% \begin{variable}[no-user-doc]{\l_graphics_search_ext_seq}
```

Comment:
This results in an undefined label, because the outer most setting wins.

> End of definition for `\__graphics_backend_loaded:n` and `\l_graphics_search_ext_seq`. This function is documented on page ??.)

I think this is a special use case that need not be sorted in `l3doc`, hence I will push a workaround.